types/devexpress-web/README.md

```diff
@@ -2,8 +2,8 @@ DevExpress ASP.NET/MVC TypeScript definitions
 =============================================
 
 You can use these TypeScript definitions for your web projects, which contain
-DevExpress ASP.NET or MVC controls. For this, simply add  at the top of your
-code `<reference path="TypeScript/devexpress-web.d.ts" />`.
+DevExpress ASP.NET or MVC controls. For this, simply add at the top of your
+code `<reference path="index.d.ts" />`.
```

Comment:
That reference won't work unless you're in the same directory as `index.d.ts`. You probably meant `/// <reference types="devexpress-web" />`.
We don't publish custom READMEs, so anything important should be moved to a comment in `index.d.ts` instead.